
seems postgres 15 is more strict, needs a space
between variable and following AND


Index: temp/puppetlabs/puppetdb/scf/storage.clj
--- temp/puppetlabs/puppetdb/scf/storage.clj.orig
+++ temp/puppetlabs/puppetdb/scf/storage.clj
@@ -1169,7 +1169,7 @@
                 {:latest_report_id report-id
                  :latest_report_timestamp producer-timestamp}
                 [(str "certname = ?"
-                      "AND ( latest_report_timestamp < ?"
+                      " AND ( latest_report_timestamp < ?"
                       "      OR latest_report_timestamp is NULL )")
                  node producer-timestamp]))
 
